internal/cephfs/controllerserver.go

```diff
@@ -794,7 +794,10 @@ func (cs *ControllerServer) CreateSnapshot(
 	}
 
 	if parentVolOptions.BackingSnapshot {
-		return nil, status.Error(codes.InvalidArgument, "cannot snapshot a snapshot-backed volume")
+		err = store.AddSnapshotBackedVolumeRef(ctx, parentVolOptions, cs.ClusterName, cs.SetMetadata, req.GetSecrets())
+		if err != nil {
+			return nil, status.Error(codes.Internal, err.Error())
+		}
```

Comment:
When there is no error, eit this function with a success result?

Reply:
Done